yarn.lock

```diff
@@ -980,6 +980,8 @@ aproba@^1.0.3, aproba@^1.1.1:
   resolved "https://registry.yarnpkg.com/aproba/-/aproba-1.2.0.tgz#6802e6264efd18c790a1b0d517f0f2627bf2c94a"
   integrity sha512-Y9J6ZjXtoYh8RnXVCMOU/ttDmk1aBjunq9vO0ta5x85WDQiQfUF9sIPBITdbiiIVcBo03Hi3jMxigBtsddlXRw==
 
+<<<<<<< HEAD
+=======
```

Comment:
This doesn't look right, please copy and paste the `yarn.lock` file from the Oppia repo.

Reply:
@safwan20 This is not fixed.

Reply:
@safwan20 Still not fixed. Run the server locally on this branch in order to regenerate this file.